documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -130,32 +276,38 @@ defines the log retention policy.
 |4
 |`topic.creation.default.compression.type` is mapped to the {link-kafka-docs}/#compression.type[`compression.type`]
 property of the {link-kafka-docs}/#topicconfigs[topic level configuration parameters] and
-defines how messages are compressed on harddisk.
+defines how messages are compressed on hard disk.
 |===
 
-As you can see, you can use every {link-kafka-docs}/#topicconfigs[topic level configuration parameter]
-as property.
-
 [NOTE]
 ====
-Note that `replication.factor` and `partitions` properties are mandatory for the `default` group
-and optional for custom groups. Custom groups will fallback to the `default` group's value if not
-set. Use `-1` to use the Kafka broker's default value.{empty} +
-This fallback does not apply to other config parameters.
+Custom groups fall back to the `default` group settings only for the required `replication.factor` and `partitions` properties.
+If the configuration for a custom topic group leaves other properties undefined, the values specified in the `default` group are not applied.
 ====
 
+// Type: procedure
+// ModuleID: specifying-the-configurations-for-debezium-custom-topic-creation-groups
+// Title: Specifying the configuration for {prodname} custom topic creation groups
+[id="custom-topic-creation-group-configuration"]
 === Custom group configuration
 
-You can specify multiple groups. Similar to the `default` group you group properties together by
-the group name. This will look like that in your connector JSON:
+You can define multiple custom topic groups, each with its own configuration.
 
+.Procedure
+ifdef::community[]
+* To define a custom topic group, add a `topic.creation._<group_name>_.include` property to the connector JSON, and
+list the properties for the custom group after the group name.
++
+The following example shows sample configurations for the `inventory` and `applicationlogs` custom topic creation groups:
++
 [source,options="nowrap",json]
 ----
 {
     ...
 
     //<1>
     "topic.creation.inventory.include": "dbserver1\\.inventory\\.*",  //<2>
+    "topic.creation.inventory.replication.factor": 3,
```

Comment:
I think I left out that `replication.factor` property on purpose to showcase the above note, that `Custom groups fall back to the `default` group settings for the required "replication.factor" and "partitions" properties`

Reply:
Oh, ok. Yup, I made a mistake when adding the CR example, and inadvertently overwrote the original JSON example. Bad copy/paste error when I went to restore it, which I then propagated to the completed example later. :(